main_test.go

```diff
@@ -233,6 +233,77 @@ func TestController(t *testing.T) {
 	}
 }
 
+func TestEndpointGenerationWithEmptyCluterName(t *testing.T) {
```

Comment:
Like this @bwplotka ?

![am_i_doing_this_right](https://user-images.githubusercontent.com/7171250/107157053-3f15f280-6982-11eb-89b1-b758ac83f406.jpg)